### PR TITLE
Replace admin guide links

### PR DESCRIPTION
**Pull request by Zhao-Andy**
## What type of PR is this? (check all applicable)
- [x] Documentation Update

## Description
New admin guide, who dis?

## QA Instructions, Screenshots, Recordings
- [ ] Check that all the new links work

### UI accessibility concerns?
Nope

## Added tests?
- [x] No, none needed

## [Forem core team only] How will this change be communicated?

- [x] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post

## [optional] Are there any post deployment tasks we need to perform?
Nope

### CODE REVIEW

File: app/controllers/admin/application_controller.rb [hunk 1]
```diff
@@ -5,18 +5,18 @@ class ApplicationController < ApplicationController
...
-      navigation_links: "https://forem.gitbook.io/forem-admin-guide/admin/navigation-links",
-      pages: "https://forem.gitbook.io/forem-admin-guide/admin/pages",
-      podcasts: "https://forem.gitbook.io/forem-admin-guide/admin/podcasts",
-      reports: "https://forem.gitbook.io/forem-admin-guide/admin/reports",
-      users: "https://forem.gitbook.io/forem-admin-guide/admin/users",
-      html_variants: "https://forem.gitbook.io/forem-admin-guide/admin/html-variants",
-      display_ads: "https://forem.gitbook.io/forem-admin-guide/admin/display-ads",
-      chat_channels: "https://forem.gitbook.io/forem-admin-guide/admin/chat-channels",
-      tags: "https://forem.gitbook.io/forem-admin-guide/admin/tags"
+      badges: "https://admin.forem.com/admin/badges",
```
Comment (djuber) [inline app/controllers/admin/application_controller.rb:8]: https://admin.forem.com/admin/badges is a 404 for me?  As is https://admin.forem.com/badges without the /admin/ path component.

**Review by djuber — CHANGES_REQUESTED** [review]
still checking but it looks like the help urls in the admin controller might not be ready.

File: app/views/admin/overview/index.html.erb [hunk 2]
```diff
@@ -8,17 +8,17 @@
...
-          <li class="mb-1 -ml-4">Browse the <a href="https://forem.gitbook.io/forem-admin-guide" data-action="click->ahoy#trackOverviewLink">Forem Admin Guide</a></li>
+          <li class="mb-1 -ml-4">Set up and configure your Forem under <%= link_to "Config", admin_config_path, "data-action": "click->ahoy#trackOverviewLink" %></li>
+          <li class="mb-1 -ml-4"><%= link_to "Invite people", admin_invitations_path, "data-action": "click->ahoy#trackOverviewLink" %> to join your Forem</li>
+          <li class="mb-1 -ml-4"><%= link_to "Update your Forem tags", admin_tags_path, "data-action": "click->ahoy#trackOverviewLink" %></li>
+          <li class="mb-1 -ml-4"><%= link_to "Create pages", admin_pages_path, "data-action": "click->ahoy#trackOverviewLink" %> and <%= link_to "sort them in the navigation sidebar", admin_navigation_links_path, "data-action": "click->ahoy#trackOverviewLink" %></li>
+          <li class="mb-1 -ml-4">Read the <a href="https://admin.forem.com/quick-start-guide" data-action="click->ahoy#trackOverviewLink">Quick Start Guide</a></li>
```
Comment (djuber) [inline app/views/admin/overview/index.html.erb:15]: I think this is /docs/quick-start-guide (the last changed line in the PR was good).

File: app/controllers/admin/application_controller.rb (hunk 1 above)
Comment (djuber) [reply app/controllers/admin/application_controller.rb]: I see - /docs/quick-start-guide/ and /docs/admin/badges!

Comment (djuber) [reply app/controllers/admin/application_controller.rb]: so I double checked the new values from the last commit and I get a 404 for d/a/users and d/a/config only now.


```shell
urls=("https://admin.forem.com/docs/admin/badges"
 "https://admin.forem.com/docs/admin/badges"
 "https://admin.forem.com/docs/admin/config"
 "https://admin.forem.com/docs/admin/navigation-links"
 "https://admin.forem.com/docs/admin/pages"
 "https://admin.forem.com/docs/admin/podcasts"
 "https://admin.forem.com/docs/admin/reports"
 "https://admin.forem.com/docs/admin/users"
 "https://admin.forem.com/docs/admin/html-variants"
 "https://admin.forem.com/docs/admin/display-ads"
 "https://admin.forem.com/docs/admin/chat-channels"
 "https://admin.forem.com/docs/admin/tags")

for url in ${urls[@]} do
  echo "$url" $(curl -I -s "$url" | grep HTTP)
done

https://admin.forem.com/docs/admin/badges HTTP/2 301 
https://admin.forem.com/docs/admin/badges HTTP/2 301 
https://admin.forem.com/docs/admin/config HTTP/2 404 
https://admin.forem.com/docs/admin/navigation-links HTTP/2 301 
https://admin.forem.com/docs/admin/pages HTTP/2 301 
https://admin.forem.com/docs/admin/podcasts HTTP/2 301 
https://admin.forem.com/docs/admin/reports HTTP/2 301 
https://admin.forem.com/docs/admin/users HTTP/2 404 
https://admin.forem.com/docs/admin/html-variants HTTP/2 301 
https://admin.forem.com/docs/admin/display-ads HTTP/2 301 
https://admin.forem.com/docs/admin/chat-channels HTTP/2 301 
https://admin.forem.com/docs/admin/tags HTTP/2 301 
```

Comment (Zhao-Andy) [reply app/controllers/admin/application_controller.rb]: Fixed, thanks for being more thorough than me 😅

Comment (djuber) [reply app/controllers/admin/application_controller.rb]: https://admin.forem.com/docs/admin/config/README 
https://admin.forem.com/docs/admin/users/README

Looks like topics with child pages land on path/README and don't automatically redirect from path/ to path/README?

**Comment by Zhao-Andy** [comment]
@djuber dunno what happened but I guess GitHub didn't render my changes 🤔 should be good now.

**Comment by djuber** [comment]
> @djuber dunno what happened but I guess GitHub didn't render my changes thinking should be good now.

yep - you committed 17 minutes ago and I just now see it! Async is hard :man_shrugging:

**Review by djuber — APPROVED** [review]

**Review by citizen428 — APPROVED** [review]

**Review by rhymes — COMMENTED** [review]
Ok this is a last minute nitpick, feel free to ignore, but shouldn't the domains be:

- admins.forem.com
- developers.forem.com

Also, if it's in a subdomain, can't we map the docs to the root? `admins.forem.com/docs/podcasts` instead of `/docs/admin/podcasts` ?

**Comment by Zhao-Andy** [comment]
> Ok this is a last minute nitpick, feel free to ignore, but shouldn't the domains be:
> - admins.forem.com
> - developers.forem.com

@rhymes admin.forem.com is set up already unfortunately, but we technically could change it to admins.forem.com. I'll ask Community Success what they think!

docs.forem.com is the canonical URL. If we want to change it this would be a good time to decide.

> Also, if it's in a subdomain, can't we map the docs to the root? admins.forem.com/docs/podcasts instead of /docs/admin/podcasts ?

Ah this is because the /admin/ is a section/folder for those specific pages on the docs. We can change it to something like /admin-tools/ or something

**Pull request merged**